src/main/java/com/github/manolo8/darkbot/modules/utils/NpcAttacker.java

```diff
@@ -144,9 +144,11 @@ private boolean shouldSab() {
 
     private boolean shouldRsb() {
         if (!main.config.LOOT.RSB.ENABLED || !target.npcInfo.extra.has(NpcExtra.USE_RSB)) return false;
-        if (usedRsb < System.currentTimeMillis() - main.config.LOOT.RSB.AMMO_REFRESH) usedRsb = System.currentTimeMillis();
+        boolean isReady = bar.findItemById("ammunition_laser_rsb-75").map(i -> i.activatable).orElse(false);
 
-        return usedRsb > System.currentTimeMillis() - 50;
+        if (isReady && usedRsb < System.currentTimeMillis() - 2000) usedRsb = System.currentTimeMillis();
+
+        return isReady && usedRsb > System.currentTimeMillis() - 100;
```

Comment:
```suggestion
        if (isReady && usedRsb < System.currentTimeMillis() - 1000) usedRsb = System.currentTimeMillis();
        return usedRsb > System.currentTimeMillis() - 50;
```
Let's keep the min-selection timeout the same (50ms) as it seemed to work right, and let's lower the min delay of activation here. There's no reason to keep it that high when it will be activated almost instantly